docs/tempo/website/setup/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Set up a Tempo cluster
+menuTitle: Set up a Tempo cluster
+description: Learn how to set up a Tempo cluster and visualize data
+weight: 150
+---
+
+# Set up a Tempo cluster
```

Comment:
Can we make this a single, numbered task?

Reply:
Done, but at a higher level. There will be additional information for these steps that we'll need to add later.